benchmarks/opmphm.c

```diff
@@ -0,0 +1,60 @@
+#include "benchmarks.h"
+
+static uint32_t * getRandomSeed (uint32_t * seed);
+
+static size_t shapef (size_t initSize ELEKTRA_UNUSED, size_t size, int level, uint32_t * seed ELEKTRA_UNUSED)
+{
+	if (!size) return 0;
+	if (level > 3) return 0;
+	return 1;
+}
+
+int main (int argc ELEKTRA_UNUSED, char ** argv ELEKTRA_UNUSED)
+{
+	uint32_t seed;
+	if (getRandomSeed (&seed) != &seed)
+	{
+		fprintf (stderr, "FATAL: Seed Parsing Error or feed me more seeds\n");
+		return EXIT_FAILURE;
+	}
+	KeySetShape shape;
+	shape.parent = 1;
+	shape.minWordLength = 3;
+	shape.maxWordLength = 7;
```

Comment:
What about command-line arguments for the seed and the shape? (If given, otherwise use some default)